documentation/feature/weaponselect.md

```diff
@@ -7,27 +7,38 @@ parent: wiki
 
 ## Overview
 
-### Weapon select
-The number key can be used to quickly switch between weapons. (1 key - pistol, 2 key - rifle, 3 key - grenade launcher, 4 key - rocket launcher, 5 key - binocular)
-
 ### Holster weapon
-Adds the ability to holster a weapon on the back. (0 key)
+Adds the ability to holster a weapon on the back.
 
-### Engine select
-Quickly turn engine on and off (1 key - turn off, 2 key - turn on)
+### Quick weapon select
+Adds key bindings to quickly switch weapons while on foot or in a vehicle. (Unbound by default).
 
-### Weapon select
-Quickly switch between vehicle weapons (1-3 key)
+### Quick vehicle engine on/off
+Adds key bindings to quickly turn a vehicle engine on or off.
 
 ### Grenade select
-To prevent accidents a grenade has to be selected before it can be thrown. Toggles between explosive and non-explosive grenades. When spamming the throw key, the player won't automatically switch to frag grenades when all smokes are used up. Also shows an indicator to quickly see how many grenades are left when selecting and after throwing (6 key - switch between frag grenades, 7 key - switch between other grenades)
-
+Changes the grenade selection key bindings to help prevent accidents.
 
 ## Usage
 
-Short overview of how to use the feature, e.g. menu options, key bindings, 
-instructions. May not apply to all modules.
+### Weapons
+
+Press <kbd>0</kbd> to holster and unholster a weapon. When a weapon is holstered, the weapon fire button will not pull the weapon out. This can be used as an additional level of safety or as a roleplaying feature.
```

Comment:
Both forms are fine. Source: am avid dungeons and dragons player
On May 14, 2015 11:25 PM, "ulteq" <notifications@github.com> wrote:

> In documentation/feature/weaponselect.md
> <https://github.com/acemod/ACE3/pull/1210#discussion_r30385748>:
>
> >
> >  ## Usage
> >
> > -Short overview of how to use the feature, e.g. menu options, key bindings,
> > -instructions. May not apply to all modules.
> > +### Weapons
> > +
> > +Press <kbd>0</kbd> to holster and unholster a weapon. When a weapon is holstered, the weapon fire button will not pull the weapon out. This can be used as an additional level of safety or as a roleplaying feature.
>
> I think it should be role playing in this case, but I'm not sure.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/acemod/ACE3/pull/1210/files#r30385748>.
>